code/game/objects/items/devices/radio/radio.dm

```diff
@@ -74,6 +74,16 @@
 	/// associative list of the encrypted radio channels this radio can listen/broadcast to, of the form: list(channel name = channel frequency)
 	var/list/secure_radio_connections
 
+	/// overlay when speaker is on
+	var/overlay_speaker_idle = "s_idle"
+	/// overlay when recieving a message
+	var/image/overlay_speaker_active = "s_active"
```

Comment:
This shouldn't be typed as image since this is a string.

Reply:
You have not fixed this.

Reply:
i was working on the flick_overlay pr then i took a break to work on something else please dont close this pr